### PR TITLE
Enable CodeBuild to fetch base image from ECR.

### PR DESCRIPTION
**Pull request by fajrinazis**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -132,6 +132,7 @@ resource "aws_codebuild_project" "ci" {
     compute_type = "${var.compute_type}"
     image        = "${var.image}"
     type         = "LINUX_CONTAINER"
+    image_pull_credentials_type = "${var.bake_codebuild_image_credentials}"
```
Comment (salvianreynaldi) [inline main.tf:135]: please change the variable name to `image_credentials`, or `image_pull_credentials_type`, as `bake` is not relevant for terraform ci/cd image, and can be confusing

**Review by febryantonius — APPROVED** [review]
~LGTM~ my bad, should update the variable name as Sal mentioned above first

File: variables.tf [hunk 2]
```diff
@@ -36,6 +36,12 @@ variable "image" {
   description = "Docker image used by CodeBuild"
 }
 
+variable "bake_codebuild_image_credentials" {
```
Comment (febryantonius) [inline variables.tf:39]:
```suggestion
variable "image_credentials" {
```

File: main.tf (hunk 1 above)
Comment (febryantonius) [inline main.tf:135]:
```suggestion
    image_pull_credentials_type = "${var.image_credentials}"
```

**Comment by andysaputra** [comment]
What about the default image variable?

**Review by febryantonius — APPROVED** [review]

**Pull request merged**